modules/apps/document-library/document-library-preview-pdf/src/main/resources/META-INF/resources/document_library_preview_pdf_settings/js/PDFPreviewLimit.js

```diff
@@ -22,11 +22,14 @@ const PDFPreviewLimit = ({maxLimitSize, namespace, scopeLabel, value}) => {
 	const [inputValue, setInputValue] = useState(value);
 
 	const onChange = (event) => {
-		const value = event.target.value;
+		const value = parseInt(event.target.value, 10);
```

Comment:
The second parameter of `parseInt` is not needed. It is 10 by default.

Reply:
SF rule, parameter needed